mitosheet/src/mito/components/endo/FormulaBar.tsx

```diff
@@ -89,7 +89,7 @@ const FormulaBar = (props: {
                         mitoContainerRef={props.mitoContainerRef}
                     />
                 } 
-                {props.editorState?.editorLocation !== 'formula bar' &&
+                {props.editorState?.type!=="closed" && props.editorState?.editorLocation !== 'formula bar' &&
```

Comment:
Again, non-required ?. I'll stop commenting on them -- but can you go through and remove them if they aren't needed? I think they are almost needed no where now - and not having them makes a line way more readable!